app/views/admin/podcasts/edit.html.erb

```diff
@@ -33,7 +33,7 @@
             <li class="list-group-item w-100">
               <%= link_to "@#{owner.username}", user_url(owner), target: "_blank", rel: :noopener %>
               <%# f.hidden_field :user_id, value: owner.id %>
-              <%# f.submit "Remove", class: "btn btn-danger btn-sm float-right" %>
+              <%# f.submit "Remove", class: "crayons-btn crayons-btn--danger crayons-btn--s float-right" %>
```

Comment:
Although this was not important right now, I did remove bootstrap classes from comments too to be on safer side.